torchaudio/prototype/functional/functional.py

```diff
@@ -0,0 +1,42 @@
+import math
+
+import torch
+
+
+def convolve(x: torch.Tensor, y: torch.Tensor, use_fft: bool = False) -> torch.Tensor:
```

Comment:
I feel like FFT-version can be a stand alone function. WDYT?

Reply:
split it out into two functions